nautobot/core/settings.py

```diff
@@ -105,26 +105,38 @@
 RACK_ELEVATION_DEFAULT_UNIT_HEIGHT = 22
 RACK_ELEVATION_DEFAULT_UNIT_WIDTH = 220
 
-# Remote auth
+# Global 3rd-party authentication settings
+EXTERNAL_AUTH_BACKENDS = []
+EXTERNAL_AUTH_DEFAULT_GROUPS = []
+EXTERNAL_AUTH_DEFAULT_PERMISSIONS = {}
+
+# Remote auth backend settings
 REMOTE_AUTH_AUTO_CREATE_USER = False
-REMOTE_AUTH_DEFAULT_GROUPS = []
-REMOTE_AUTH_DEFAULT_PERMISSIONS = {}
-REMOTE_AUTH_ENABLED = True  # FIXME(jathan): Deprecated in Nautobot
+# Remove in favor of EXTERNAL_AUTHxx
+# REMOTE_AUTH_DEFAULT_GROUPS = []
+# REMOTE_AUTH_DEFAULT_PERMISSIONS = {}
+# REMOTE_AUTH_ENABLED = True  # FIXME(jathan): Deprecated in Nautobot
 REMOTE_AUTH_HEADER = "HTTP_REMOTE_USER"
 
+# SSO backend settings
+# SOCIAL_AUTH_ENABLED = False # Deprecated
+# Remove in favor of EXTERNAL_AUTHxx
+# SOCIAL_AUTH_MODULE = ""
+# SOCIAL_AUTH_DEFAULT_GROUPS = []
+# SOCIAL_AUTH_DEFAULT_PERMISSIONS = {}
+# Removing in favor or proper group management
+# SOCIAL_AUTH_DEFAULT_STAFF = False
+# SOCIAL_AUTH_DEFAULT_SUPERUSER = False
+SOCIAL_AUTH_POSTGRES_JSONFIELD = False
+# If we're not going to namespace settings, be clear on who owns what
+NB_SOCIAL_AUTH_URL_NAMESPACE = "sso"
```

Comment:
Let's please expand this prefix from `NB_` to `NAUTOBOT_`.

Reply:
I will remove the commented lines, but wanted to leave them in the draft for visibility and clarity of my intent.